client/components/wcc-settings-form/settings-group.js

```diff
@@ -57,6 +47,26 @@ const SettingsGroup = ( props ) => {
 			);
 
 		case 'step':
+			if ( undefined !== form.acceptSuggestion ) {
```

Comment:
True, we just generally have used the typeof way -- TIL that this (comparing directly to undefined) is safe since ECMAScript 5